src/screens/About/About.tsx

```diff
@@ -44,14 +60,31 @@ export const sidebarItems: TocItem[] = [
         label: aboutContent.futureProjectsHeader,
       },
       {
-        to: '/about#join-us',
-        label: aboutContent.joinUsHeader,
+        to: '/about#contact-us',
+        label: aboutContent.contactUsHeader,
       },
     ],
   },
 ];
 
 // TODO(Chelsi): put these ids into the CMS
+const Button: React.FC<{ to?: string; href?: string; cta: string }> = ({
+  to,
+  href,
+  cta,
+}) => {
```

Comment:
Consider adding a comment to tell that we should pass either `to` or `href`, but not both. I don't think we need to assert for that though (`LargeOutlineButton` should trigger a TS error if we do pass both)